pkg/printers/internalversion/printers.go

```diff
@@ -1155,8 +1165,24 @@ func printJob(obj *batch.Job, options printers.GenerateOptions) ([]metav1.TableR
 	default:
 		jobDuration = duration.HumanDuration(obj.Status.CompletionTime.Sub(obj.Status.StartTime.Time))
 	}
+	var status string
+	if obj.ObjectMeta.DeletionTimestamp != nil {
+		status = "Terminating"
+	} else if hasJobCondition(obj.Status.Conditions, batch.JobComplete) {
+		status = "Complete"
+	} else if hasJobCondition(obj.Status.Conditions, batch.JobFailed) {
+		status = "Failed"
+	} else if hasJobCondition(obj.Status.Conditions, batch.JobSuspended) {
+		status = "Suspended"
+	} else if hasJobCondition(obj.Status.Conditions, batch.JobFailureTarget) {
```

Comment:
@mimowo do you think this one is useful for a user?

Reply:
I'm on a fence here. Terminating pods in reaction to FailJob action might take ~30s between SIGTERM and SIGKILL. Knowing that the job is doomed to be failed might be useful I guess.

Reply:
Okay. I'm happy with it then.